chain/default_syncer.go

```diff
@@ -367,3 +370,21 @@ func (syncer *DefaultSyncer) HandleNewBlocks(ctx context.Context, blkCids []cid.
 	}
 	return nil
 }
+
+// IsReorg determines if a new tipset would cause a reorg if heavier than the
```

Comment:
any tipset would cause a reorg if heavier than the syncers head. i think you mean if would cause a re org?

Reply:
This is not true if the new tipset is a child of the syncer's head.